### PR TITLE
added values to categoryEnum

### PR DESCRIPTION
**Pull request by Allensy**

### CODE REVIEW

File: test/karma-unit.conf.js [hunk 1]
```diff
@@ -44,8 +44,9 @@ module.exports = function (config) {
             'bower_components/Chart.js/Chart.js',
             'bower_components/angular-chart.js/dist/angular-chart.js',
             'bower_components/v-accordion/dist/v-accordion.js',
-            'bower_components/satellizer/satellizer.js',
+            'bower_components/satellizer/dist/satellizer.js',
```
Comment (assafshp) [inline test/karma-unit.conf.js]: ....???? check you have the latest versio

File: test/karma-unit.conf.js [hunk 2]
```diff
@@ -44,8 +44,9 @@ module.exports = function (config) {
...
             'bower_components/angular-drag-and-drop-lists/angular-drag-and-drop-lists.js',
+            'bower_components/angular-ui-grid/ui-grid.js',
```
Comment (assafshp) [inline test/karma-unit.conf.js]: why ?

File: src/components/configSat/decorator/categoryService.decorator.js [hunk 3]
```diff
@@ -17,7 +17,12 @@
                 ['STRATEGY', 8, 'strategy'],
                 ['SUBJECT', 9, 'subject'],
                 ['SUB_SCORE', 10, 'subScore'],
-                ['TEST_SCORE', 11, 'testScore']
+                ['TEST_SCORE', 11, 'testScore'],
+                ['LEVEL1', 9, 'level1Categories'],
```
Comment (assafshp) [inline src/components/configSat/decorator/categoryService.decorator.js]: I not sure it is good to have the same number in enum

**Review by assafshp — APPROVED** [review]

File: src/components/configSat/decorator/categoryService.decorator.js (hunk 3 above)
Comment (Allensy) [inline src/components/configSat/decorator/categoryService.decorator.js]: It's ok when you want to access it by name

File: test/karma-unit.conf.js (hunk 1 above)
Comment (Allensy) [inline test/karma-unit.conf.js]: yes

File: test/karma-unit.conf.js (hunk 2 above)
Comment (Allensy) [inline test/karma-unit.conf.js]: 'bower install' installed it

**Pull request merged**